### PR TITLE
Re-style welcome and login pages

### PR DESCRIPTION
**Pull request by martinbedouret**
image

image

### CODE REVIEW

**Review by sylvansson — CHANGES_REQUESTED** [review]
Love the new background 👌

**Review by kidcompassion — CHANGES_REQUESTED** [review]
This looks great, I love the animation! 😻 

The only issue I see is that the welcome panel is getting clipped, which seems to be coming from the padding on `.WelcomeScreen`, which then pushes the panel down, where it gets clipped by the `overflow:hidden` set on body, html. I _think_ if you turn it off, it'll work. 💯 

Screen Shot 2020-05-20 at 9 50 34 PM

With padding: 
Screen Shot 2020-05-20 at 9 53 20 PM

Without padding:
Screen Shot 2020-05-20 at 9 49 52 PM

**Review by sylvansson — APPROVED** [review]

File: src/components/AuthScreen/AuthScreen.component.js [hunk 1]
```diff
@@ -1,120 +1,27 @@
...
-            isDialogOpen={activeView === 'forgot'}
-            onClose={this.resetActiveView}
-          />
-          <SignUp
-            isDialogOpen={activeView === 'signup'}
-            onClose={this.resetActiveView}
-          />
-        </div>
-      </Dialog>
+      <div className="AuthScreen__content">
```
[the remark this answers is not in the document]
Comment (martinbedouret) [reply src/components/AuthScreen/AuthScreen.component.js:16]: Yes , I fixed it. thank you!

**Comment by martinbedouret** [comment]
Thank you guys for the feedback! Appreciate

**Pull request merged**